docs/kafka/kcat-kafka/README.adoc

```diff
@@ -73,8 +73,7 @@ ifndef::community[]
 endif::[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 * You have a running Kafka instance in {product-kafka}.
-* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* https://adoptopenjdk.net/[JDK^] 11 or later is installed. The latest LTS version of OpenJDK is recommended.
```

Comment:
Need parentheses for consistency: `(The latest ... )`

Reply:
Thanks Stetson. Will correct.